juwai/juwai-lint-cfg
# [improve-frontend] collaborate new juwai-com-v2

## Summary of changes
- removed .jshintrc
- added .eslintrc
- changed .jscsrc
## How to Test
- Please use [juwai-com-v2#improving-frontend](https://github.com/juwai/juwai-com-v2/pull/269) to test this PR.
- Please make sure your [juwai-com-v2#improving-frontend](https://github.com/juwai/juwai-com-v2/pull/269) has changed package.json of "juwai-lint-cfg" to "git+https://github.com/monochrome-yeh/juwai-lint-cfg.git#improving-frontend".

/fyi: @arkhi , @HouCoder : Please review.
